cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -653,4 +653,11 @@ export default {
   verifyInstanceDetailsViewAbsent() {
     cy.expect(instanceDetailsSection.absent());
   },
+
+  searchByStatus(status) {
```

Comment:
method doesn't correspond to action. Need to click checkbox, not to verify that it's exists